plugins/woocommerce/src/Blocks/BlockTypes/BlockifiedProductDetails.php

```diff
@@ -24,6 +89,47 @@ class BlockifiedProductDetails extends AbstractBlock {
 	 * @return string Rendered block output.
 	 */
 	protected function render( $attributes, $content, $block ) {
-		return $content;
+		$parsed_block = $block->parsed_block;
+
+		if ( is_admin() || WC()->is_rest_api_request() ) {
+			return $content;
+		}
+
+		/**
+		 * Filter the product tabs in the product details block.
+		 *
+		 * @since 3.3.0
+		 * @param array $tabs Array of product tabs.
+		 */
+		$product_tabs = apply_filters(
+			'woocommerce_product_tabs',
+			array()
+		);
+
+		$product_tabs_without_native_tabs = array_filter(
+			$product_tabs,
+			function ( $tab ) {
+				$default_tab = array( 'description', 'reviews', 'additional_information' );
+				return ! in_array( $tab, $default_tab, true );
+			},
+			ARRAY_FILTER_USE_KEY
+		);
+
+		$parsed_tabs_added_via_hook = array_reduce(
+			$product_tabs_without_native_tabs,
+			function ( $carry, $tab ) {
+				ob_start();
+				call_user_func( $tab['callback'] );
```

Comment:
I think we should also pass in the tab index key and the tab object to this callback function, in the same way it is called within Woo currently: https://github.com/woocommerce/woocommerce/blob/3765a665d52b6302ca20db50bb0592b88cd4f3af/plugins/woocommerce/templates/single-product/tabs/tabs.php#L47
Some plugins depend on these arguments and would break if called this way.

Reply:
Great catch! Fixed with c9a6c25c055d37005266a8293f9621fb2c9cb3da